package.json

```diff
@@ -26,9 +26,11 @@
   "homepage": "https://github.com/18F/cg-site#readme",
   "devDependencies": {
     "browserify": "^13.1.0",
+    "mermaid": "^7.1.0",
     "node-sass": "^3.10.0",
     "onchange": "git+https://git@github.com/msecret/onchange.git",
-    "postcss-import": "^8.1.2"
+    "postcss-import": "^8.1.2",
```

Comment:
I see that we're importing `mermaid`, but where do we use `postcss-import` or `scuttle`?

Reply:
so I believe, that `postcss-import` was already there. I'm hoping to publish Scuttle `1.1.0` which will include the compiled CSS in the package so we can copy it out the same way that `cg-style` is copied into the project from `node_modules`. We can remove it for now though, it's not being used at the moment.

Reply:
in other words, I'm not really sure where we use `postcss-import` 😁

It might be good to clean things up, but I don't want to turn this into a huge amount of work to refactor my way out of this frontend paper bag.

Reply:
OK, seems like we can leave `scuttle` out until installing it actually does something.